src/devices/Dhtxx/Devices/Dht22.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Device.Gpio;
+using Iot.Units;
+
+namespace Iot.Device.DHTxx
+{
+    /// <summary>
+    /// Temperature and Humidity Sensor DHT22
+    /// </summary>
+    public class Dht22 : DhtBase
+    {
+        /// <summary>
+        /// Create a DHT22 sensor
+        /// </summary>
+        /// <param name="pin">The pin number (GPIO number)</param>
+        /// <param name="pinNumberingScheme">The GPIO pin numbering scheme</param>
+        public Dht22(int pin, PinNumberingScheme pinNumberingScheme = PinNumberingScheme.Logical)
+            : base(pin, pinNumberingScheme)
+        { }
+
+        internal override double GetHumidity(byte[] readBuff)
+        {
+            return (readBuff[0] << 8 | readBuff[1]) * 0.1;
+        }
+
+        internal override Temperature GetTemperature(byte[] readBuff)
```

Comment:
are those all the same?

Reply:
No, because I don't have all the sensors, I overlooked this issue in previous PRs. So `GetTemperature` needs to be override.

Reply:
If the code is the same please create some base class for them or just inherit one from another